plugin/evm/vm.go

```diff
@@ -337,7 +338,22 @@ func (vm *VM) Initialize(
 	_ []*commonEng.Fx,
 	appSender commonEng.AppSender,
 ) error {
-	vm.extensionConfig = &extension.Config{}
+	// TODO: this is to prevent migration of VM tests
+	// however this is not the intended change
+	// and should be removed after everything is migrated to atomicvm.VM
+	// atomicVM.Initialize() should be calling this vm's Initialize()
```

Comment:
Am I correct in understanding that this means the `*atomicvm.VM` will be the "primary" VM that is ultimately presented to consensus?

I can't remember exactly what they were, but Stephen recently told me about some concerns with VM wrapping. @StephenButtolph do you think they could be issues here?

Reply:
Please reference the PR number in the TODO comment so the path to resolution is clear.